SECURITY.md

```diff
@@ -0,0 +1,61 @@
+# Security policy
+
+## Charmcraft and charms
+
+Charmcraft is a tool to create and publish charms. A charm is a software artifact that
+contains instructions to manage a separate application in the context of a [Juju] model,
```

Comment:
```suggestion
Charmcraft is a tool to create and publish charms. A component of the [Juju]
ecosystem, a charm is a piece of software that orchestrates and manages software
services,
```

Trying to simplify/clarify this part. @lengau Thoughts on this wording?

Reply:
I'm fine either way.